2_process.R

```diff
@@ -100,11 +100,33 @@ p2_targets_list <- list(
   # Extract Road Salt raster values to catchments polygons in the DRB - general function raster_to_catchment_polygons
   tar_target(
     p2_rdsalt_per_catchment,
-    {lapply(p1_rd_salt, function(x) raster_to_catchment_polygons(polygon_sf = p1_catchments_sf_valid,
+    {lapply(p1_rdsalt, function(x) raster_to_catchment_polygons(polygon_sf = p1_catchments_sf_valid,
                                                                     raster = x, categorical_raster = FALSE,
-                                                                    raster_summary_fun = sum, new_cols_prefix = 'rd_slt'))
+                                                                    raster_summary_fun = sum, new_cols_prefix = 'rd_slt', , na.rm = T))
```

Comment:
There are 2 commas in a row here. Was that needed to make this work? Or can one be deleted?